docs/en/ingest-management/commands.asciidoc

```diff
@@ -669,6 +670,45 @@ elastic-agent version
 <hr>
 ++++
 
+[discrete]
```

Comment:
Same comment as before. Move this to the top section because it appears first alphabetically.